tensorpipe/channel/cuda_basic/context.cc

```diff
@@ -47,13 +47,13 @@ class Context::Impl : public Context::PrivateIface,
   ~Impl() override = default;
 
  private:
-  std::string domainDescriptor_;
```

Comment:
This is no longer stored, and instead we forward `cpuContext_->domainDescriptor()` where needed.